ludwig/utils/image_utils.py

```diff
@@ -27,6 +28,19 @@
 from ludwig.constants import CROP_OR_PAD, INTERPOLATE
 from ludwig.utils.data_utils import get_abs_path
 from ludwig.utils.fs_utils import get_bytes_obj_from_path
+from ludwig.utils.registry import Registry
+
+TVModelVariant = namedtuple("TVModelVariant", "variant_id create_model_function weights_class")
```

Comment:
Instead of a `namedtuple`, could you make this a dataclass, which also allows for type hints?

https://stackoverflow.com/questions/34269772/type-hints-in-namedtuple